deployment/common/changeset/set_config_mcms.go

```diff
@@ -0,0 +1,207 @@
+package changeset
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/config"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	commontypes "github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+type ConfigPerRole struct {
+	Proposer  config.Config
+	Canceller config.Config
+	Bypasser  config.Config
+}
+type ProposalConfig struct {
+	MinDelay time.Duration // delay for timelock worker to execute the transfers.
+}
+
+type SetConfigParams struct {
+	ConfigsPerChain map[uint64]ConfigPerRole
+	ProposalConfig  *ProposalConfig
```

Comment:
elsewhere we've been calling it MCMSConfig, could stick to that

Reply:
renamed here too https://github.com/smartcontractkit/chainlink/pull/15684/commits/0f4cbf369a6f66423b89a2ffdd68add367691c1e